olm-catalog/serverless-operator-index/image_content_source_policy.yaml

```diff
@@ -134,8 +148,6 @@ spec:
       source: "registry.redhat.io/openshift-serverless-1/serverless-openshift-kn-rhel8-operator"
     - mirrors: ["quay.io/redhat-user-workloads/ocp-serverless-tenant/serverless-operator-135/serverless-bundle"]
       source: "registry.redhat.io/openshift-serverless-1/serverless-operator-bundle"
-    - mirrors: ["quay.io/redhat-user-workloads/ocp-serverless-tenant/serverless-operator-135/serverless-operator-bundle"]
-      source: "registry.redhat.io/openshift-serverless-1/serverless-operator-bundle-rhel8"
```

Comment:
@mgencur @maschmid: here it gets cleaned up again

Reply:
Nice! It looks good now.